Content.Shared/Content.Shared.csproj

```diff
@@ -9,6 +9,7 @@
     <OutputPath>../bin/Content.Shared</OutputPath>
     <Configurations>Release;Debug</Configurations>
     <Platforms>AnyCPU</Platforms>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
This is not happening, because content has to be able to be sandboxed, and unverifiable code (which unsafe is) would break that.